<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepairServiceWidget</class>
 <widget class="QWidget" name="RepairServiceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>工作人员</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>1200</width>
     <height>660</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="uidE">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>请输入维修ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ridE">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>请输入设备ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="didE">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>请输入用户ID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ecb">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>请选择报修事件</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tcb">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>100</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>请选择时间</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="scb">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>100</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>请选择状态</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>维修服务大厅</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Agency FB&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>右键或双击数据进行编辑</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
